specs/rails/db_migrate.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: Apply database migrations
+command: |-
+    rails db:migrate
+tags:
+  - rails
+description: Uses the rails rake task for applying all the migrations
+source_url: "https://stackoverflow.com/questions/7323261/uninstall-remove-a-homebrew-package-including-all-its-dependencies"
```

Comment:
Is this source_url correct? It seems to be referencing the brew command instead of anything rails related.